article.md

```diff
@@ -3,5 +3,24 @@ This is my first article
 
 Introduction
 
+Personal information
+
 My name is lili el khalil
-Master student at UdeM
+Born in 03 august 1998, my birthday is in 4 days !!
+My status in Canada is as an International student
```

Comment:
Once again, I suggest removing this type of personal information from a public document.